tests/jasmine/test-skipped-hooks.ts

```diff
@@ -0,0 +1,7 @@
+import assert from 'node:assert'
+
+describe('Jasmine smoke test to be skipped via wdio hooks', () => {
+    it('should be skipped', async () => {
+        assert.strictEqual(1, 1)
```

Comment:
How about we let this assertion fail as we expect this test never to be executed.
```suggestion
        assert.strictEqual(1, 2)
```

Reply:
done